src/c/core/session/session.c

```diff
@@ -187,10 +187,11 @@ bool uxr_run_session_time(uxrSession* session, int timeout_ms)
     uxr_flash_output_streams(session);
 
     bool timeout = false;
-    while(!timeout)
+    do
```

Comment:
This is not necessary since `bool timeout = false; while(!timeout){...}` will execute always at least once.

Reply:
yes, i see it.